handbook/sales/roles/gtm_sales_operations.md

```diff
@@ -0,0 +1,59 @@
+# Sales Strategy & Operations Analyst or Manager
```

Comment:
Can we move this to the ops section instead, and link to it there from the Sales jobs list?

Reply:
I can do this in a follow-up PR if painful for you to do it here by the way

Reply:
I'm on it!